jobs/JREGIONAL_BIAS_CORRECTION_O3

```diff
@@ -19,6 +19,10 @@
 source_config_for_task "cpl_aqm_parm|task_run_post|task_bias_correction_o3" ${GLOBAL_VAR_DEFNS_FP}
 . $USHdir/job_preamble.sh
 #
+export DCOMROOT=${DCOMROOT:-/lfs/h1/ops/prod/dcom}
+export DCOMINairnow=${DCOMINairnow:-/lfs/h1/ops/prod/dcom}
+export COMINbicor=${COMINbicor:-$(compath.py ${NET}/${aqm_ver})}
```

Comment:
Looks to me, the default value have been defined in var_defns.sh.
grep -i "comin"  var_defns.sh |wc -l
11

Reply:
@chan-hoo I agree these items should be defined at a higher level, but often J-jobs have these kind of definitions to provide a fallback in case it hasn't been defined above.